locale/en-us/dialog/en/another_shop_locations.dialog

```diff
@@ -0,0 +1 @@
+Another locations.
```

Comment:
Consider updating to something like `This store has more locations.` or `There are more locations.`